predicators/spot_utils/integration_tests.py

```diff
@@ -315,6 +316,149 @@ def test_move_with_sampling() -> None:
+        rgbds = capture_images(robot, localizer, [hand_camera])
+
+        # Run detection to get a pixel for grasping.
+        _, artifacts = detect_objects([brush], rgbds)
+        pixel = get_object_center_pixel_from_artifacts(artifacts, brush,
+                                                       hand_camera)
+
+        # Pick at the pixel with a top-down grasp.
+        grasp_at_pixel(robot, rgbds[hand_camera], pixel)
+        localizer.localize()
+
+        # Stow the arm.
+        stow_arm(robot)
+        localizer.localize()
+
+        # Move to the bucket.
+        robot_pose = localizer.get_last_robot_pose()
+        rel_pose = get_relative_se2_from_se3(robot_pose, detections[bucket],
+                                             pre_place_nav_distance,
+                                             pre_place_nav_angle)
+        navigate_to_relative_pose(robot, rel_pose)
+        localizer.localize()
+
+        # Place in the bucket.
+        robot_pose = localizer.get_last_robot_pose()
+        surface_rel_pose = robot_pose.inverse() * detections[bucket]
+        place_rel_pos = math_helpers.Vec3(
+            x=surface_rel_pose.x + place_offset_x,
+            y=surface_rel_pose.y,
+            z=surface_rel_pose.z + place_offset_z)
+        place_at_relative_position(robot,
+                                   place_rel_pos,
+                                   downward_angle=np.pi / 2)
+        localizer.localize()
+
+        # Stow the arm again.
+        stow_arm(robot)
+        localizer.localize()
+
+        # Navigate to look at the bucket.
+        robot_pose = localizer.get_last_robot_pose()
+        rel_pose = get_relative_se2_from_se3(robot_pose, detections[bucket],
+                                             pre_dump_nav_distance,
+                                             pre_place_nav_angle)
+        navigate_to_relative_pose(robot, rel_pose)
+        localizer.localize()
+
+        # Look at the bucket.
+        move_hand_to_relative_pose(robot, DEFAULT_HAND_LOOK_FLOOR_POSE)
+        localizer.localize()
+
+        # Capture an image.
+        rgbds = capture_images(robot, localizer, [hand_camera])
+        rgbd = rgbds[hand_camera]
+
+        # Choose a grasp.
+        _, artifacts = detect_objects([bucket], rgbds)
+
+        r, c = get_object_center_pixel_from_artifacts(artifacts, bucket,
+                                                      hand_camera)
+        pixel = (r + bucket_grasp_dr, c)
+
+        # Grasp at the pixel with a top-down grasp.
+        top_down_rot = math_helpers.Quat.from_pitch(np.pi / 2)
+        grasp_at_pixel(robot, rgbd, pixel, grasp_rot=top_down_rot)
+
+        # Dump!
+        dump_container(robot, post_dump_place_offset_z)
+        localizer.localize()
+
+        # Stow the arm again.
+        stow_arm(robot)
+        localizer.localize()
+
+
 if __name__ == "__main__":
-    test_all_find_move_pick_place()
+    # test_all_find_move_pick_place()
     # test_move_with_sampling()
```

Comment:
do we really want to keep commenting and uncommenting various tests here? should we by default leave them uncommented on master?